docs/hub/endpoints.md

```diff
@@ -12,10 +12,11 @@ We have open endpoints that you can use to retrieve information from the Hub as
+| /api/datasets  <br/>  GET                                                            | Get information from all datasets in the Hub.  You can specify additional parameters to have more specific results.  <br/>- `author`: Filter datasets by an other or organization, such as `huggingface` or `microsoft`<br />- `filter`: Filter based on tags, such as `task_categories:text-classification` or `languages:en`.<br/>- `sort`: Property to use when sorting. <br/>- `direction`: Direction in which to sort. <br/>- `limit`: Limit the number of datasets fetched. <br/>- `full`: Whether to fetch most dataset data, such as all tags, the files, etc.                                                                                                                                                                                                                                                                                                                                                | `list_datasets()`                      | ```params= { "filter":"filter", "full":"full", "sort": "sort", "direction": "direction", "limit": "limit" }```                   |
 | /api/datasets/{repo_id} <br/> /api/datasets/{repo_id}/revision/{revision} <br/>  GET | Get all information for a specific dataset.  <br/>- `full`: Whether to fetch most dataset data, such as all tags, the files, etc.                                                                                                                                                                                                                                                                                                                                             | `dataset_info(repo_id, revision)`        | ```headers = { "authorization" :  "Bearer $token", "full" : "full"  }```                                                                                     |
+| /api/metrics  <br/>  GET                                                            | Get information from all metrics in the Hub.                                                                                                                                                                                                                                                                                                                                                 | `list_metrics()`                      ||
```

Comment:
My motivation with exposing it here is since there is a functional api within `HfApi` for it (e.g. `list_metrics`), it's in the docs that it doesn't take in params and just grabs them all.

Reply:
@julien-c kept it in for final, if we want it removed let me know